### PR TITLE
Added AMoAd.xml

### PR DESCRIPTION
**Pull request by luixxiul**

### CODE REVIEW

File: src/chrome/content/rules/AMoAd.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+		- tech.amoad.com   (times out)
+
+-->
+
+<ruleset name="AMoAd">
+
+	<target host="p.amoad.com" />
+	<target host="www.amoad.com" />
+	<target host="manage.amoad.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/AMoAd.xml]: j.amoad.com
amoad.com

File: src/chrome/content/rules/AMoAd.xml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<!--
+
+	Nonfunctional domains:
+
+	Problematic domains:
+
+		- tech.amoad.com   (times out)
```
Comment (J0WI) [inline src/chrome/content/rules/AMoAd.xml]: cdni.amoad.com

**Comment by J0WI** [comment]
Could you also sort the hosts alphabetically?

**Comment by luixxiul** [comment]
@J0WI ok done.

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**